### ISSUE TITLE
Boot servers using WOL

### ISSUE THREAD

**Issue by MarkusTeufelberger** [siderolabs/sidero#852]
Another option to get a server that's currently shut down back up next to IPMI (and others, see #221 #289) would be https://en.wikipedia.org/wiki/Wake-on-LAN

Not fit for all use cases (it has its limitations), but WOL could be useful for home lab environments and/or consumer hardware that doesn't contain a dedicated BMC.

**Comment by MarkusTeufelberger** [comment]
Yeah, you'd need to rely on ACPI shutdown/reboot actually working in that scenario, not all functionality of IPMI would be covered of course. It would still be an improvement to not being able to trigger a boot remotely at all, as it is currently the case.

**Comment by aarnaud** [comment]
I opened a discussion here about alternative if BMC can't be use: https://github.com/siderolabs/sidero/discussions/1307

**Comment by MarkusTeufelberger** [comment]
:-(
